packages/workflow-app-textedit/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "workflow-app-textedit",
+  "version": "1.0.0-rc.0",
+  "main": "index.js",
+  "description": "Workflow app for TextEdit",
+  "license": "MIT",
+  "keywords": [
+    "workflow-wm"
```

Comment:
We have an problem to solve with these apps. We need to make a listing of what platform/wm they support. Now I would want to either have this as annotations here in the `keywords` or some executable specification that could be autogenerated somehow. 

We could have a document writer which tries to `open` the app and listen for the `Platform is not supported` errors. If so, we need to make a package which contains this error mechanism in a more standard way than today.

Reply:
I think the platform logic errors should be moved out of the app/layout/vm and be specified with `platforms: ["osx"]` on the object. We can then use that list to generate what we need, if we need something more.

Reply:
No, I meant more like
```
{
  type: "app",
  platforms: ["osx"]
}
```

Reply:
Cool. 

Playing with the api again:
```
const {validate} from "shared/apps";

{ 
  type: "app",
  platforms: ["osx"],

  open: (..., context) => {
     validate(this, context);
  }
}
```

Reply:
I think validate should be lifted into workflow, specifying supported platforms should be enough for third party apps to get this functionality.